api/python/observation.md

```diff
@@ -0,0 +1,155 @@
+    the property value associated with returned `Observation`'s by the property
+    [`observedNode`](https://browser.datacommons.org/kg?dcid=observedNode)
+
+*   `measured_property (str)`: The property value associated with returned
+    `Observation`'s by the property
+    [`measuredProperty`](https://browser.datacommons.org/kg?dcid=measuredProperty)
+
+*   `stats_type (str)`: The statistical type of the `Observation`. This is commonly set
+    to `measuredValue`.
+
+*   `observation_date`: The property value associated with returned
+    `Observation`'s by the property
+    [`observationDate`](https://browser.datacommons.org/kg?dcid=observationDate).
+    This is specified in ISO8601 format.
+
+*   `observation_period (str)`: The property value associated with returned
+    `Observation`'s by the property
+    [`observationPeriod`](https://browser.datacommons.org/kg?dcid=observationPeriod)
+    of the observation. If the `Observation` has this property set, this must
+    be specified.
+
+*   `measurement_method (str)`: The property value associated with returned
+    `Observation`'s by the property
+    [`measurementMethod`](https://browser.datacommons.org/kg?dcid=measurementMethod)
+    of the observation. If the `Observation` has this property set, this must
+    be specified
+
+**Returns**
+
+When `dcids` is an instance of `list`, the returned `Observation`'s are
+formatted as a `dict` from a given DCID to the unique `Observation` observing
+the DCID where the observation is specified by what is given in the other
+parameters *if such exists*. A given DCID will *NOT* be a member of the `dict`
+if such an observation does not exist.
+
+When `dcids` is an instance of `pandas.Series`, the returned `Observation`'s
+are formatted as a `pandas.Series` where the `i`-th entry corresponds to
+observation observing the given DCID as specified by the other parameters *if
+such exists*. Otherwise, the cell holds NaN.
+
+**Raises**
+
+*   `ValueError` - If the payload returned by the Data Commons REST API is malformed or the API key is not set.
+
+Be sure to initialize the library, and specify the API key. Check the [Python library setup guide](/api/python/) for more details.
+
+## Examples
+
+We would like to get the following for December, 2018:
+* The [total count of employed persons in California](https://browser.datacommons.org/kg?dcid=dc/o/wetnm9026gf73)
+* The [total count of employed persons in Kentucky](https://browser.datacommons.org/kg?dcid=dc/o/4nklvdnkfq835)
+* The [total count of employed persons in Maryland](https://browser.datacommons.org/kg?dcid=dc/o/nkntbc4vpshn9>)
+
+The observations we want are observations of the populations representing
+employed individuals in each state (to get these, see
+[`get_populations`](/api/python/population.html). With a list of these
+population DCIDs, we can get the observations like so:
+
+```python
+>>> import datacommons as dc
```

Comment:
Small nit: I was thinking of separating the `>>>` prompt and output since the python code block will colorize the output dictionary.

If you want to separate the code and output, then I think we can drop the `>>>` altogether since then users can copy and paste the code snippets into their commandline directly.

Reply:
yup, i ran into that too (cpy/pste). we can update that later, maybe after the styling updates.

Reply:
That sounds like a good plan